### DISCUSSION TITLE
Improvement in the Calendar block

### DISCUSSION THREAD

**Discussion by MedVox** [quillforms/quillforms#150] [Ideas]
**Problem:**
The Calendly block has a serious problem. It does not have an option so that after scheduling it automatically moves to the next block.

It is counterintuitive to schedule in the Calendly block and still have to click on another button in the same block to move to the next block.

When I was creating my form I identified this possible problem and now that the form is running the problem is actually happening. Users are coming to the Calendly block, scheduling and then abandoning the form because they believe everything has already been completed. As it stands, the Calendly block fulfills its purpose (scheduling), but Quillforms makes the user experience difficult for them to complete the form.

**Solution:**
Could be solved with a checkbox ⬇️

**[  ] After scheduling, automatically advance to the next block**

This way I would eliminate the need for the "Already scheduled text in mobile" message field that I am using to carry alert messages like "Not long to go, move on to finish!" to improve advancement rates after scheduling.

That's it guys, I hope I helped you and I would really like the block to be improved, this improvement is essential for anyone who uses the block.

**Comment by mdmag** [comment]
This has been fixed already
